### ISSUE TITLE
What is the state of Clementine? Is it maintenance, or active development?

### ISSUE THREAD

**Issue by mcmurphy8097234789** [clementine-player/Clementine#6756]
Currently, the Clementine website has the last blogpost dated 2016. However, the github here shows various updates and tweaks every now and then. So is clementine being maintained, or is it being actively developed? I am asking because it appears that some distro developers, like those at Solus OS, do not want to include clementine in their repo because they believe development has ceased

**Comment by clementine-user** [comment]
This was discussed a while back in an older issue. It seems as though it touched something with a few folks and it woke up a bit. Unfortunately, since Clementine is being built now with GitHub CI, builds for macOS after Clementine-1.4.0rc1-211 don't run on anything less than 10.15.x.

**Comment by dmdmdm** [comment]
I think the RC (release candidate) should be released.  I use it successfully on Linux and Windows.

**Comment by phirestalker** [comment]
Is there an easy way to get it running on macOS 10.14? Version 1.31 crashes for me, so I am trying the 1.4... builds, but they require 10.15.

**Comment by clementine-user** [comment]
Unless the maintainers change the CI build scripts to build against macOS 10.13.x or 10.14.x, the answer is probably no, so you would be stuck at 1.4.0rc1-211.

**Comment by matkoniecz** [comment]
> However, the github here shows various updates and tweaks every now and then. So is clementine being maintained, or is it being actively developed?

It is in weird state of being developed, while not making any new releases. See "Please make new release" (#5958).

Though there is hope as there is a RC (release candidate) from early 2020.

**Comment by alexanderdd** [comment]
yes **please close this issue** and continue over here https://github.com/clementine-player/Clementine/issues/5958

**Comment by matkoniecz** [comment]
Well, if Clementine would not be in some weird zombie state then this issue would be closed by devs or people authorized by them to filter issue tracker.
